### PR TITLE
Remove server.Policy.

### PR DESCRIPTION
**Pull request by lthibault**
- [x] Remove `server.Policy`
- [x] Remove `*server.Policy` from call sites
- [x] Update code generator

This PR requires you to run `./regen.sh`.

### CODE REVIEW

File: server/server_test.go [hunk 1]
```diff
@@ -105,32 +105,13 @@ func (seq *callSeq) GetNumber(ctx context.Context, call air.CallSequence_getNumb
 	return nil
 }
 
-type lockCallSeq struct {
```
Comment (zenhack) [inline server/server_test.go:108]: Is this just an unrelated drive-by "oh look, this is dead code," or is this related the the primary change in this pr in some way I'm missing?

**Review by zenhack — APPROVED** [review]
LGTM.

File: server/server_test.go (hunk 1 above)
Comment (lthibault) [reply server/server_test.go:108]: Correct, it's just dead code that was flagged by the linter.  I figured it wouldn't add too much noise to the PR (and then forgot to mention it).
